.github/workflows/release-drafter.yml

```diff
@@ -19,7 +21,13 @@ jobs:
         pull-requests: write
     runs-on: ubuntu-latest
     steps:
-      # Drafts your next Release notes as Pull Requests are merged into the default branch
-      - uses: release-drafter/release-drafter@v5
+      - name: Check branch name
+        id: branch_check
+        #-qE : quiet mode Search for a regular expression pattern in the input text using extended(E) regular expressions
+        run: echo "is_release_branch=$(echo $GITHUB_REF | grep -qE 'refs/heads/(release-|master)' && echo true || echo false)" >> "$GITHUB_ENV"
+
+      - name: Draft Release
+        if: steps.branch_check.outputs.is_release_branch == 'true'
+        uses: release-drafter/release-drafter@v5
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
And what if `steps.branch_check.outputs.is_release_branch` is false? No draft release for master?

Reply:
The `if` condition above will check if the branch name starts with `release-**` or `master`. And set it to true. The condition will be false only if the branch name is something other than `release-**` or `master`. But my concern now is, we cannot create 2 Draft release notes with same tag. And what if we have both the branches present.

Reply:
got it. we can create 2 drafts with the same tag only if we have 2 branches.

this will work:
master - tagA
release - tagA

this wont work:
master - tagA
master - tagA